src/content/post/20161122_drupal_application_bottlenecks_avoid_with_profiling.md

```diff
@@ -0,0 +1,170 @@
++++
+date = "2016-11-22T11:02:30Z"
+draft = false
+title = "Avoid bottlenecks on a Drupal web application and how identify it with profiling"
+description = "How identify bottleneck of your web application and avoid that can appear in the future"
+tags        = [ "profiling", "drupal", "performance", "web application"  ]
+topics      = [ "Profiling", "Drupal", "Performance" ]
+author = "Vinz"
++++
+
+## Introdution
+
+This article is focused on the possible *bottlenecks of a web application*, how you can identify it and improve the performance of your code.
+
+We'll talk about using *Blackfire* to get insights on a Drupal-based PHP application running on a LAMP stack.
+
+<!-- more -->
+
+Application's performance is one of the feature that tells a *done* job from a *good* job, and shows a focus on quality.
+
+Scope of this post it to provide a quick-reference can be used right before to deploy a Drupal (but not only) project to the production.
```

Comment:
Very good, just some grammar-nazism at work :D 

* *that* can be used
*  right before to deploy -> right before deploying (people are strange in UK)
* to the production -> to production (they are strange and use articles where we would not :P )